packages/ui/src/index.ts

```diff
@@ -140,9 +140,6 @@ export type {
 export { default as Form } from './components/molecules/Form'
 export type { FormProps } from './components/molecules/Form'
 
-export { default as Alert } from './components/molecules/Alert'
-export type { AlertProps } from './components/molecules/Alert'
-
```

Comment:
it's right?

Reply:
yep, me remove it from ui and move it to components :)